torchvision/prototype/datasets/_builtin/stanford_cars.py

```diff
@@ -0,0 +1,85 @@
+from typing import Any, Dict, List
+
+from torchdata.datapipes.iter import Filter, IterDataPipe, Mapper, Zipper
+from torchvision.prototype.datasets.utils import Dataset, DatasetConfig, DatasetInfo, HttpResource, OnlineResource
+from torchvision.prototype.datasets.utils._internal import hint_sharding, hint_shuffling, path_comparator, read_mat
+from torchvision.prototype.features import BoundingBox, EncodedImage
+
+
+class StanfordCars(Dataset):
+    def _make_info(self) -> DatasetInfo:
+        return DatasetInfo(
+            name="stanford_cars",
```

Comment:
For now the convention is to use a hyphen to separate words in the dataset name.
```suggestion
            name="stanford-cars",
```

Note that you will also have to rename the `.categories` file and should rename the data folder `~/.cache/torch/datasets/vision/stanford_cars` to avoid re-downloading everything.